sensor/common/scannerv4definitions/repomappings/http_handler.go

```diff
@@ -0,0 +1,105 @@
+package repomappings
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+	"sync/atomic"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/clientconn"
+	"github.com/stackrox/rox/pkg/httputil"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/mtls"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/sensor/common"
+	"google.golang.org/grpc/codes"
+)
+
+const repoMappingPath = "/api/extensions/scanner-v4/repomappings"
+
+var (
+	headersToProxy = set.NewFrozenStringSet("If-Modified-Since", "Accept-Encoding")
+	log            = logging.LoggerForModule()
+)
+
+// Handler handles requests to retrieve scanner repository mapping data
+// from Central.
+type Handler struct {
+	centralClient    *http.Client
+	centralReachable atomic.Bool
+}
+
+// NewRepoMappingHandler creates a new scanner repository mapping data handler.
+func NewRepoMappingHandler(centralEndpoint string) (*Handler, error) {
+	client, err := clientconn.NewHTTPClient(mtls.CentralSubject, centralEndpoint, 0)
+	if err != nil {
+		return nil, errors.Wrap(err, "instantiating central HTTP transport")
```

Comment:
`fmt.Errorf()` is recommended since `pkg/errors` is archived.

Reply:
I decide to leave the sensor part to another PR, what do you think?

Reply:
Your comment is not related to my suggestion, maybe supposed to be a reply to another comment?

Reply:
Yeah, I see you copied and pasted the same reply elsewhere.